javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,103 @@
+let name = [];
+let todoList = [];
+function getReply(command) {
+  if (command.toLowerCase().includes("hello my name is")) {

```

Comment:
You could call `command.toLowerCase()` once at the top and save it into a variable, to avoid repeating it on every call